components/aws/sagemaker/deploy/README.md

```diff
@@ -0,0 +1,62 @@
+# SageMaker Hosting Services - Create Endpoint Kubeflow Pipeline component
```

Comment:
This file header should be the only single hash `#`. This will help with README reader tools

Reply:
got it, done